packages/app/src/runs/RunsContainer.vue

```diff
@@ -27,10 +27,11 @@
       data-cy="runs"
       class="flex flex-col pb-[24px] gap-[16px]"
     >
-      <Warning
+      <TrackedWarning
         v-if="userProjectStatusStore.cloudStatusMatches('needsRecordedRun') && userProjectStatusStore.project.isUsingGit"
         :title="t('runs.empty.noRunsFoundForBranch')"
         :message="noRunsForBranchMessage"
+        banner-id="aci_052023_noRunsFoundForBranch"
```

Comment:
Add new banner IDs to `BannerIds` map in `packages/types/src/config.ts`

Reply:
Ah, didn't know these existed. Thank you

Reply:
Import the values from the BannerId variables you added